### PR TITLE
Bug 1121274 - Blur the input field before deleting the IME

### PR DESCRIPTION
**Pull request by janjongboom**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[janjongboom Jan Jongboom (janjongboom)](https://github.com/janjongboom) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=b09e1c61f827)

File: apps/keyboard/test/marionette/uninstallation_test.js [hunk 1]
```diff
@@ -88,6 +88,12 @@ marionette('Show Keyboard App after uninstallation', function() {
     keyboard.imeSwitchingKey.click();
     check3rdPartyIme();
 
+    // Blur the input field, otherwise the keyboard frame has higher
```
Comment (RudyLu) [inline apps/keyboard/test/marionette/uninstallation_test.js]: Please add XXX: here to note that this is a workaround in the test.

**Comment by RudyLu** [comment]
@janjongboom 
Please help correct the pull request title and commit message.
Thanks.

**Comment by try-server-hook** [comment]
[janjongboom Jan Jongboom (janjongboom)](https://github.com/janjongboom) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=7eb825e92cc1)

**Pull request merged**